controller/environment.go

```diff
@@ -132,3 +143,17 @@ func (c *EnvironmentController) Show(ctx *app.ShowEnvironmentContext) error {
 	}
 	return ctx.OK(res)
 }
+
+func (c *EnvironmentController) checkClustersUser(ctx context.Context, clusterURL string) error {
+	clusters, err := c.clusterService.UserClusters(ctx)
```

Comment:
FYI, if can be simplified in the future, once https://github.com/fabric8-services/fabric8-cluster/issues/59 is done

Reply:
We can't use `/api/clusters` with URL as we need clusters who linked to users.  For ex, we call `/api/clusters` with URL and we get response with cluster_details.  It just mean that the cluster is a valid cluster and available with cluster_service but it can't guarantee that the cluster is linked to user also.

Reply:
@nurali-techie ok, my bad indeed! In that case, would you need a new endpoint such as `/api/user/cluster?url=...` to get a single response (or ` 404` if the user is not linked to the cluster with the given URL)?

Reply:
kool, so we don't have such a endpoint today.

Reply:
right, but if you need it, please open an issue and I'll take care of it